systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -60,6 +60,8 @@ public interface Constants {
     // it is replacement instead of checking logs for reconciliation using dynamic waiting on some change for some period of time
     int GLOBAL_RECONCILIATION_COUNT = (int) ((RECONCILIATION_INTERVAL / GLOBAL_POLL_INTERVAL) + GLOBAL_STABILITY_OFFSET_COUNT);
 
+    long THROTTLING_EXCEPTION_TIMEOUT = Duration.ofMinutes(10).toMillis();
```

Comment:
Seems rather long. What is the reason for it?

Reply:
Actual implementation of the admin clients block the thread until some timeout, which can be, f.e. 5 minutes (we need to improve this in the next PR to the test-clients).

Currently, there is 5 minutes timeout for the throttling quota exception, which can be easily hit and the test then fails, even though the job fails right after 2 seconds. I added there 10 minutes just to be 100% sure.

After we fix the admin client behavior, I wanted to decrease this timeout.

Maybe some comment here would be good.

Reply:
@scholzj 
> Even five minutes seem to be like awfully lot of waiting.

Yeah, it's awful for me as well, but without rewriting the whole admin client, I cannot do anything with it..

Reply:
Ok :-(